programs/svm-spoke/src/instructions/slow_fill.rs

```diff
@@ -225,6 +225,10 @@ pub fn execute_v3_slow_relay_leaf(
     // Emit the FilledV3Relay event
     let message_clone = relay_data.message.clone(); // Clone the message before it is moved
 
+    if message_clone.len() > 0 {
+        invoke_handler(ctx.accounts.signer.as_ref(), ctx.remaining_accounts, &message_clone)?;
```

Comment:
`execute_v3_slow_relay_leaf` will use more transaction size than `fill_v3_relay`, due to the proof. This might further limit the complexity of the across+ style hooks that can be used depending on whether they were executed fast or slow.